python/requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 # Dev requirements
 gitpython
 httpretty
-mypy
+mypy<0.990
```

Comment:
Is there a reason not to use `=`, if I'm reading the version specifier this says do not allow pre-releases, is that correct?

Reply:
it just means use the latest version that's below 0.990. so 0.990 is the upper bound. I just don't know how mypy releases things and it appears they don't do semantic versioning so this is just to cover scenarios if they do a future 0.8x release